test/unit/libraries/EscrowState.t.sol

```diff
@@ -0,0 +1,322 @@
+        Duration minAssetsLockDuration = Durations.from(minAssetsLockSeconds);
+
+        _context.minAssetsLockDuration = minAssetsLockDuration;
+
+        Vm.Log[] memory entries = vm.getRecordedLogs();
+
+        EscrowState.setMinAssetsLockDuration(_context, minAssetsLockDuration);
+
+        checkContext(State.NotInitialized, minAssetsLockDuration, D0, D0, T0);
+
+        assertEq(entries.length, 0);
+    }
+
+    // ---
+    // checkSignallingEscrow()
+    // ---
+
+    function test_checkSignallingEscrow_happyPath() external {
+        _context.state = State.SignallingEscrow;
+        EscrowState.checkSignallingEscrow(_context);
+    }
+
+    function test_checkSignallingEscrow_RevertOn_InvalidState() external {
+        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedState.selector, State.SignallingEscrow));
+
+        EscrowState.checkSignallingEscrow(_context);
+    }
+
+    // ---
+    // checkRageQuitEscrow()
+    // ---
+
+    function test_checkRageQuitEscrow_happyPath() external {
+        _context.state = State.RageQuitEscrow;
+        EscrowState.checkRageQuitEscrow(_context);
+    }
+
+    function test_checkRageQuitEscrow_RevertOn_InvalidState() external {
+        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedState.selector, State.RageQuitEscrow));
+
+        EscrowState.checkRageQuitEscrow(_context);
+    }
+
+    // ---
+    // checkBatchesClaimingInProgress()
+    // ---
+
+    function test_checkBatchesClaimingInProgress_happyPath() external view {
+        EscrowState.checkBatchesClaimingInProgress(_context);
+    }
+
+    function testFuzz_checkBatchesClaimingInProgress_RevertOn_InvalidState(
+        uint40 rageQuitExtensionDelayStartedAtSeconds
+    ) external {
+        vm.assume(rageQuitExtensionDelayStartedAtSeconds > 0);
+        _context.rageQuitExtensionDelayStartedAt = Timestamps.from(rageQuitExtensionDelayStartedAtSeconds);
+        vm.expectRevert(EscrowState.ClaimingIsFinished.selector);
+
+        EscrowState.checkBatchesClaimingInProgress(_context);
+    }
+
+    // ---
+    // checkWithdrawalsTimelockPassed()
+    // ---
+
+    function testFuzz_checkWithdrawalsTimelockPassed_happyPath(
+        uint16 rageQuitExtensionDelayStartedAtSeconds,
+        uint16 rageQuitExtensionDelaySeconds,
+        uint16 rageQuitWithdrawalsTimelockSeconds
+    ) external {
+        vm.assume(rageQuitExtensionDelayStartedAtSeconds > 0);
+
+        Duration rageQuitExtensionDelay = Durations.from(rageQuitExtensionDelaySeconds);
+        Duration rageQuitWithdrawalsTimelock = Durations.from(rageQuitWithdrawalsTimelockSeconds);
+
+        _context.rageQuitExtensionDelayStartedAt = Timestamps.from(rageQuitExtensionDelayStartedAtSeconds);
+        _context.rageQuitExtensionDelay = rageQuitExtensionDelay;
+        _context.rageQuitWithdrawalsTimelock = rageQuitWithdrawalsTimelock;
+
+        vm.warp(
```

Comment:
I'm not sure that it will make the test easier to understand, take a look pls at these lines
https://github.com/lidofinance/dual-governance/pull/92/files#diff-94006bb4d7eaa8960b0a0a1fd4de0d02c0d50140ca8857d0ba68052b5aeee11fR209-R216
However I replaced all `vm.warp()` calls with `_wait()` as you requested.